### ISSUE TITLE
Trans component to translate the alt text from images

### ISSUE THREAD

**Issue by pcorpet** [i18next/react-i18next#1032]
**Is your feature request related to a problem? Please describe.**
I sometimes use text that uses image to replace some of the words, those images have an alt text so that they stay readable if the image is not visible, and I end up translating separately the global sentence and the image's alt text.

```jsx
<Trans>
An image is worth <img src={myImage} alt="a thousand words" />
</Trans>
```

which requires to translate the strings:
```json
"An image is worth <0></0>": "Un schéma vaut mieux qu'<0></0>",
"a thousand words": "un long discours"
```

**Describe the solution you'd like**
I'd love to handle the image as a text wrapper as I do with other types:
```json
"An image is worth <0>a thousand words</0>": "Un schéma vaut mieux qu'<0>un long discours</0>"
```
and that react-i18next updates the alt text of the image accordingly.

**Describe alternatives you've considered**
Create my own image component that converts the `children` prop in `alt` prop.

```jsx
<Trans>
An image is worth <Image src={myImg}>a thousand words</Image>
</Trans>
```

**Comment by jamuhl** [comment]
The problem `An image is worth <0>a thousand words</0>` from where do we know that we need to fill the alt attribute and not the title attribute or children on the element at that place?

**Comment by pcorpet** [comment]
It would work only on img components which alt props is specifically meant for that (replace the component by text)

**Comment by jamuhl** [comment]
If we do something - it should cover all cases (attributes)...so my current suggestion is going for the alternative you already mentioned (converting the children to alt prop)
